.travis.yml

```diff
@@ -10,8 +10,9 @@ before_install:
 - tar xf .travis/secret_keys.tar -C .travis
 - mv .travis/travis ~/.ssh/id_rsa
```

Comment:
We use .travis/travis-ci-aws-frank.pem in our builds
```transport:
 +  ssh_key: '.travis/travis-ci-aws-frank.pem'
```
in .kitchen.cloud.yml

Can you remove this moving operation?